packages/components/src/toolbar/index.js

```diff
@@ -54,11 +56,12 @@ function Toolbar( { controls = [], children, className } ) {
 	}
 
 	return (
-		<div className={ classnames( 'components-toolbar', className ) }>
+		<ToolbarContainer className={ classnames( 'components-toolbar', className ) }>
 			{ flatMap( controlSets, ( controlSet, indexOfSet ) => (
 				controlSet.map( ( control, indexOfControl ) => (
-					<div
+					<ToolbarButtonContainer
 						key={ [ indexOfSet, indexOfControl ].join() }
+						keyProp={ [ indexOfSet, indexOfControl ].join() }
```

Comment:
Why do we repeat key and keyProp?

Reply:
Ah, good catch, I should have mentioned this "gotcha" in the PR description. So, React has this set of "special" props that it doesn't propagate down to the child components. They call them [Special Props](https://reactjs.org/warnings/special-props.html). In this case, the `key` property cannot be picked up by the `ToolbarButtonContainer` component and we need to replicate it on purpose into a differently named prop, `keyProp`.